bedrock/base/templates/base-resp.html

```diff
@@ -167,13 +167,36 @@
     </div><!-- close #outer-wrapper -->
 
     <!--[if IE 9]>
-      <script src="{{ static('js/libs/matchMedia.js') }}"></script>
+      {% javascript 'matchmedia' %}
```

Comment:
Nested conditional comments don't seem to work, meaning pages that have a conditional addition of `matchmedia.addListener` break the new logic in `base-resp.html`. `base-new.html` has both `matchMedia` and `matchMedia.addListener` bundled, which I'm totally in favor of and implementing here.